pkg/util/mountOpts.go

```diff
@@ -112,6 +112,14 @@ func ProcessOptions(options []string, isTmpfs bool, sourcePath string) ([]string
 				return nil, errors.Wrapf(ErrDupeMntOption, "only one of 'z' and 'Z' can be used")
 			}
 			foundZ = true
+		case "U":
+			if isTmpfs {
+				return nil, errors.Wrapf(ErrBadMntOption, "the 'U' option is not allowed with tmpfs mounts")
```

Comment:
Why Not?  Does this happen automatically?  IE If I am running as a user, do the tmpfs get mounted as owned by that user?

Reply:
This was not happening automatically. To add correct ownership the uid and gid will be added to the mount options when the U flag is specified